ui/src/app/utils/access-utils.tsx

```diff
@@ -188,16 +187,7 @@ export const getAccessModuleConfig = (
       AccessModule.RASLINKLOGINGOV,
       () => ({
         ...apiConfig,
-        isEnabledInEnvironment:
-          enableRasLoginGovLinking || enforceRasLoginGovLinking,
-
-        // override these API config values temporarily
-        // when we complete RW-7862, enforceRasLoginGovLinking will work as normal access enable flag
-        // and we can remove this override
-
-        requiredForRTAccess: enforceRasLoginGovLinking,
```

Comment:
these values are set in the API, but were overridden here as a special case.  We don't need to do this anymore.